src/Symfony/Component/Form/AbstractExtension.php

```diff
@@ -24,35 +24,31 @@ abstract class AbstractExtension implements FormExtensionInterface
      *
      * @var FormTypeInterface[] An array of FormTypeInterface
      */
-    private $types;
+    private array $types;
 
     /**
      * The type extensions provided by this extension.
      *
-     * @var FormTypeExtensionInterface[] An array of FormTypeExtensionInterface
+     * @var FormTypeExtensionInterface[][] An array of FormTypeExtensionInterface
```

Comment:
let's remove the comment, it's useless and wrong, isn't it?

Reply:
Done in #42110.